docs/index.asciidoc

```diff
@@ -96,6 +94,8 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-user>> |<<string,string>>|no
+| <<plugins-{type}s-{plugin}-password>> |<<password,password>>|No
```

Comment:
I don't see it link to the password option? Once we render the page it will link to this page https://www.elastic.co/guide/en/logstash/current/configuration-file-structure.html#password

Similar to the `truststore_password` options  see:

```
| <<plugins-{type}s-{plugin}-truststore_password>> |<<password,password>>|No

```